sktime/utils/plotting.py

```diff
@@ -11,7 +11,36 @@
 from sktime.utils.validation.forecasting import check_y
 
 
-def plot_series(*series, labels=None, markers=None):
+def check_pred_int(pred_int):
+    """helper function to check pred_int data type
+
+
+    pred_int: pd.DataFrame
+        Prediction intervals of series
+
+
+    Riases
+    -------
+    TypeError: when pred_int is not a pd.DataFrame
+    Exception: when the number of columns is less or more than 2
+        and column labels are not ['lower', 'upper']
+    """
+    _check_soft_dependencies("pandas")
```

Comment:
why are we checking if pandas is a soft dependency?

Reply:
The condition to check for the data type raised an error since the pd.DataFrame was not defined or import. So, I had to make sure that the pandas package was imported for this reason.

Reply:
yes but it's not a soft dependency, just import it at the top of the file without using `_check_soft_dependencies`